coderd/audit.go

```diff
@@ -189,6 +190,11 @@ func convertAuditLog(dblog database.GetAuditLogsOffsetRow) codersdk.AuditLog {
 	_ = json.Unmarshal(dblog.Diff, &diff)
 
 	var user *codersdk.User
+	var agent string
+
+	if dblog.UserAgent.Valid {
+		agent = dblog.UserAgent.String
+	}
```

Comment:
I don't think this should be necessary. You can access `dblog.UserAgent.String` safely without checking if it's valid, it'll just be an empty string, so both outcomes are essentially the same.